velox/functions/prestosql/aggregates/tests/ApproxDistinctTest.cpp

```diff
@@ -432,5 +433,14 @@ TEST_F(ApproxDistinctTest, toIntermediate) {
       digests, {"c0"}, {"merge(a0)"}, {"c0", "cardinality(a0)"}, {input});
 }
 
+TEST(ApproxDistinctSignatureTest, hyperLogLog) {
```

Comment:
As mentioned earlier, test name should match file name. Please, change to TEST_F(ApproxDistinctTest, ...

The test name 'hyperLogLog' is not very descriptive. Consider, typeRegistration.

It could be that changing to ApproxDistinctTest will end up registering scalar functions in the SetUp, in which case, this test won't reproduce the problem. I'm thinking that, perhaps, we don't need a test for this fix at all. Seems more trouble then worth it.

Reply:
Yes actually I had changed it because when I used the file name as the test name I got this error when running the test:
```
velox/cmake-build-debug/_deps/gtest-src/googletest/src/gtest.cc:2520: Failure
Failed
All tests in the same test suite must use the same test fixture
class, so mixing TEST_F and TEST in the same test suite is
illegal.  In test suite ApproxDistinctTest,
test groupByIntegers is defined using TEST_F but
test hyperLogLog is defined using TEST.  You probably
want to change the TEST to TEST_F or move it to another test
case.
```
Apologies for not mentioning this earlier in the comment. 

Yes, I agree a test doesn't seem suitable for this fix, reverted the test.